### DISCUSSION TITLE
Μέρος Β παραδοχές για παραλαβή από θυρίδα ή υπάλληλο

### DISCUSSION THREAD

**Discussion by ghost** [sdy60/syllabus#24] [Q&A] [answered]
Κύριε @epidrome, στο πρώτο link που έχετε δώσει ως παράδειγμα (https://cleverpoint.gr/en/home/) δε τίθεται θέμα εφαρμογής διανομέα, γιατί η εταιρεία δεν έχει διανομείς, δεν είναι εταιρεία κούριερ. Επίσης, αν έχω καταλάβει σωστά δεν υπάρχουν ούτε θυρίδες. Είναι εφαρμογή όπου απλά στέλνει κάποιος το δέμα του μέσω άλλου κούριερ ας πούμε στο ψιλικατζίδικο της γειτονιάς, το οποίο είναι συμβεβλημένο με το cleverpoint, και ο ψιλικατζής το παραλαμβάνει για λογαριασμό του τελικού πελάτη.
Καταλαβαίνω σωστά ότι είναι ασύμβατο παράδειγμα σε σύγκριση με την υπόλοιπη εκφώνηση ή κάνω λάθος;
Ευχαριστώ

**Comment by epidrome** [comment, marked as answer]
ισχύει η εκφώνηση, γενικά όλα τα παραδείγματα-σύνδεσμοι είναι ενδεικτικά (π.χ., εργαλεία για κατασκευή υποδείγματος), το παράδειγμα αναφέρεται στην ιδέα ότι η παραλαβή γίνεται από κάποιο σημείο, τόσο ο κούριερ, όσο και ο τελικός πελάτης σίγουρα κάποια εφαρμογή χρησιμοποιούν, πράγματι δεν έχει θυρίδα αυτή η υπηρεσία, για την ώρα τουλάχιστον..

γενικά οι ασκήσεις δεν έχουν μια σωστή απάντηση και η ορθότητα της απάντησης εξαρτάται περισσότερο από τις παραδοχές που θα γίνουν, π.χ., μπορείτε να υποθέσετε ότι δεν θα υπάρχει θυρίδα, αλλά η συναλλαγή θα γίνει από τον υπάλληλο, πάντως και σε αυτήν την περίπτωση κάποια (επιπλέον) εφαρμογή θα χρειαστεί για να καταχωρηθεί η συναλλαγή.
